crates/bevy_render/src/camera/mod.rs

```diff
@@ -18,7 +22,9 @@ impl Plugin for CameraPlugin {
         app.register_type::<Camera>()
             .register_type::<Viewport>()
             .register_type::<Option<Viewport>>()
-            .register_type::<WindowOrigin>()
+            .register_type::<Visibility>()
```

Comment:
This change looks unrelated.

Reply:
I don't see this in blame. Probably just some git weirdness.